services/app-api/forms/wp.json

```diff
@@ -891,55 +891,21 @@
                   ],
                   "text": ""
                 },
-                {
```

Comment:
hmmm, I don't think this merge resolution did what I wanted it to do... i will be making another update